tests/validators/test_float.py

```diff
@@ -105,24 +105,44 @@ def test_float_kwargs(py_and_json: PyAndJson, kwargs: Dict[str, Any], input_valu
 @pytest.mark.parametrize(
     'multiple_of,input_value,error',
     [
-        (0.5, 0.5, None),
-        (0.5, 1, None),
+        # Test cases for multiples of 0.5
+        *[(0.5, round(i * 0.5, 1), None) for i in range(-4, 5)],
+        (0.5, 0.49, Err('Input should be a multiple of 0.5')),
         (0.5, 0.6, Err('Input should be a multiple of 0.5')),
-        (0.5, 0.51, Err('Input should be a multiple of 0.5')),
+        (0.5, -0.75, Err('Input should be a multiple of 0.5')),
         (0.5, 0.501, Err('Input should be a multiple of 0.5')),
         (0.5, 1_000_000.5, None),
         (0.5, 1_000_000.49, Err('Input should be a multiple of 0.5')),
+        (0.5, int(5e10), None),
+        # Test cases for multiples of 0.1
+        *[(0.1, round(i * 0.1, 1), None) for i in range(-10, 11)],
         (0.1, 0, None),
-        (0.1, 0.0, None),
-        (0.1, 0.2, None),
-        (0.1, 0.3, None),
-        (0.1, 0.4, None),
-        (0.1, 0.5, None),
```

Comment:
Why remove any of the existing tests?

Reply:
I'm sorry I didn't make it clear enough.
The existing tests you mentioned (0.0, 0.2, 0.3, 0.4, 0.5) are now covered by the new, more comprehensive test case:

`*[(0.1, round(i * 0.1, 1), None) for i in range(-10, 11)]`

This new case not only includes all the previous values but also extends the test coverage to include a wider range of values (from -1.0 to 1.0 in 0.1 increments), covering both positive and negative numbers.